app/src/backend/powershell.ts

```diff
@@ -0,0 +1,21 @@
+import {ChildProcess, exec} from 'child_process';
+
+import {CommandCallback} from './types';
+
+const EXE = 'powershell.exe';
+const DEFAULT_ARGS = [
+  '-ExecutionPolicy',
+  'Bypass',
+  '-NoProfile',
+  '-NonInteractive',
+  '-Command',
+];
+
+const escapeCommand = (s: string): string => s.replace(/"/g, '\\"');
```

Comment:
Feels like there is the capacity for this to be non-exhaustive. Not sure if there is some builtin that would be better? Regardless, we might not hit that in practice. If you feel good about it, I'm fine with signing off.

Reply:
Since this just shells out to PowerShell, it should be assumed that anything you pass it behaves as if you're writing it in the prompt, meaning it's up to the caller to escape anything. Automatic escaping of double quotes is only necessary because the argument for the command has to be in double quotes.